napari/__main__.py

```diff
@@ -24,6 +24,44 @@ def __call__(self, *args, **kwargs):
         # prevent unrelated INFO logs when doing "napari --info"
         logging.basicConfig(level=logging.WARNING)
         print(sys_info())
+        from .plugins import plugin_manager
+
+        errors = plugin_manager.get_errors()
+        if errors:
+            names = {e.plugin_name for e in errors}
+            print("\n‼️  Errors were detected in the following plugins:")
+            print("(Run 'napari --plugin-info -v' for more details)")
+            print("\n".join([f"  - {n}" for n in names]))
+        sys.exit()
+
+
+class PluginInfoAction(argparse.Action):
```

Comment:
Doesn't need to be for this PR, but sometime soon it would be nice to pull all our actions PluginInfoAction, InfoAction, CitationAction into their own files in an actions folder somewhere